topology/New Mexico State University/High Performance Computing/SITE.yaml

```diff
@@ -0,0 +1,15 @@
+LongName: New Mexico State University Discover Cluster
+Description: Discovery cluster at NMSU
+
+ID: 12346
```

Comment:
...apparently putting 12345 in the template was dumb of me because it confused next_site_id. I'll fix the template; in the meantime, could you use 10239 instead?

Reply:
Updated